<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OptionsForm</class>
 <widget class="QDialog" name="OptionsForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>783</width>
    <height>396</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>DejaVu Sans</family>
    <pointsize>8</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Options</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,4">
   <item>
    <widget class="QTreeWidget" name="optionsTree">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <attribute name="headerVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>1</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>2</string>
      </property>
      <property name="foreground">
       <brush brushstyle="NoBrush">
        <color alpha="0">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </property>
     </column>
     <item>
      <property name="text">
       <string>Preferences</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Behavior</string>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Startup</string>
      </property>
      <property name="text">
       <string>2</string>
      </property>
      <item>
       <property name="text">
        <string>Service</string>
       </property>
       <property name="text">
        <string>3</string>
       </property>
      </item>
     </item>
     <item>
      <property name="text">
       <string>Connection</string>
      </property>
      <property name="text">
       <string>4</string>
      </property>
      <item>
       <property name="text">
        <string>Proxy</string>
       </property>
       <property name="text">
        <string>5</string>
       </property>
      </item>
     </item>
     <item>
      <property name="text">
       <string>Log settings</string>
      </property>
      <property name="text">
       <string>6</string>
      </property>
      <item>
       <property name="text">
        <string>Backup Log</string>
       </property>
       <property name="text">
        <string>7</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>System log</string>
       </property>
       <property name="text">
        <string>8</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Log processing</string>
       </property>
       <property name="text">
        <string>9</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>HTML colors</string>
       </property>
       <property name="text">
        <string>10</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Viewers</string>
       </property>
       <property name="text">
        <string>11</string>
       </property>
      </item>
     </item>
     <item>
      <property name="text">
       <string>Reports</string>
      </property>
      <property name="text">
       <string>12</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Msg Windows</string>
      </property>
      <property name="text">
       <string>13</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Mailer settings</string>
      </property>
      <property name="text">
       <string>14</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Pagers</string>
      </property>
      <property name="text">
       <string>15</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ICQ</string>
      </property>
      <property name="text">
       <string>16</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Jabber</string>
      </property>
      <property name="text">
       <string>17</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>SMS GSM</string>
      </property>
      <property name="text">
       <string>18</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>SMS SMPP</string>
      </property>
      <property name="text">
       <string>19</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Ping/Trace options</string>
      </property>
      <property name="text">
       <string>20</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>RCI</string>
      </property>
      <property name="text">
       <string>21</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Active RMA Server</string>
      </property>
      <property name="text">
       <string>22</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Scheduler</string>
      </property>
      <property name="text">
       <string>23</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Miscellaneous</string>
      </property>
      <property name="text">
       <string>24</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QTabWidget" name="PageControl">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="locale">
        <locale language="English" country="UnitedStates"/>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="pagePreferences">
        <attribute name="title">
         <string>Preferences</string>
        </attribute>
        <layout class="QVBoxLayout" name="loPreferences">
         <item>
       <widget class="SDPO::PreferencesWidget" name="wPreferences" />
         </item>
         </layout>
         </widget>
       <widget class="QWidget" name="pageBehavior">
        <attribute name="title">
         <string>Behavior</string>
        </attribute>
        <layout class="QVBoxLayout" name="loBehavior">
         <item>
       <widget class="SDPO::BehaviorWidget" name="wBehavior" />
         </item>
         </layout>
       </widget>
       <widget class="QWidget" name="pageStartupOption">
        <attribute name="title">
         <string>Startup Option</string>
        </attribute>
        <layout class="QVBoxLayout" name="loStartupOptions">
         <item>
       <widget class="SDPO::StartupOptionsWidget" name="wStartupOptions" />
         </item>
         </layout>
       </widget>
       <widget class="QWidget" name="pageService">
        <attribute name="title">
         <string>Service</string>
        </attribute>
        <layout class="QVBoxLayout" name="loService">
         <item>
         <widget class="SDPO::ServiceOptionsWidget" name="wService" />
         </item>
         </layout>
       </widget>
       <widget class="QWidget" name="pageAccessMethod">
        <attribute name="title">
         <string>Access method</string>
        </attribute>
        <layout class="QVBoxLayout" name="loAccessMethod">
         <item>
         <widget class="SDPO::AccessMethodOptionsWidget" name="wAccessMethod" />
         </item>
         </layout>
       </widget>
       <widget class="QWidget" name="pageProxy">
        <attribute name="title">
         <string>Proxy</string>
        </attribute>
        <layout class="QVBoxLayout" name="loProxy">
         <item>
         <widget class="SDPO::ProxyOptionsWidget" name="wProxy" />
         </item>
         </layout>
       </widget>
       <widget class="QWidget" name="pagePrimaryLog">
        <attribute name="title">
         <string>Primary log</string>
        </attribute>
        <layout class="QVBoxLayout" name="loPrimaryLog">
         <item>
         <widget class="SDPO::PrimaryLogOptionsWidget" name="wPrimaryLog" />
         </item>
         </layout>
       </widget>
       <widget class="QWidget" name="pageBackupLog">
        <attribute name="title">
         <string>Backup log</string>
        </attribute>
        <layout class="QVBoxLayout" name="loBackupLog">
         <item>
         <widget class="SDPO::BackupLogOptionsWidget" name="wBackupLog" />
         </item>
         </layout>
       </widget>
       <widget class="QWidget" name="pageSystemLog">
        <attribute name="title">
         <string>System log</string>
        </attribute>
        <layout class="QVBoxLayout" name="loSystemLog">
         <item>
         <widget class="SDPO::SystemLogOptionsWidget" name="wSystemLog" />
         </item>
         </layout>
       </widget>
       <widget class="QWidget" name="pageLogProcessing">
        <attribute name="title">
         <string>Log processing</string>
        </attribute>
        <layout class="QVBoxLayout" name="loLogProcessing">
         <item>
         <widget class="SDPO::LogProcessingOptionsWidget" name="wLogProcessing" />
         </item>
         </layout>
       </widget>
       <widget class="QWidget" name="HTMLPage">
        <attribute name="title">
         <string>HTML Colors</string>
        </attribute>
        <layout class="QVBoxLayout" name="loHtmlColors">
         <item>
         <widget class="SDPO::HtmlColorsOptionsWidget" name="wHtmlColors" />
         </item>
         </layout>
       </widget>
       <widget class="QWidget" name="ViewersPage">
        <attribute name="title">
         <string>Viewers</string>
        </attribute>
        <layout class="QVBoxLayout" name="loViewers">
         <item>
         <widget class="SDPO::ViewersOptionsWidget" name="wViewers" />
         </item>
         </layout>
       </widget>
       <widget class="QWidget" name="ReportsPage">
        <attribute name="title">
         <string>Reports</string>
        </attribute>
        <layout class="QVBoxLayout" name="loReports">
         <item>
         <widget class="SDPO::ReportsOptionsWidget" name="wReports" />
         </item>
         </layout>
       </widget>
       <widget class="QWidget" name="pageMsgWindows">
        <attribute name="title">
         <string>Msg Windows</string>
        </attribute>
        <layout class="QVBoxLayout" name="loMsgWindowOptions">
         <item>
         <widget class="SDPO::MsgWindowOptionsWidget" name="wMsgWindow" />
         </item>
         </layout>
       </widget>
       <widget class="QWidget" name="pageMailerSettings">
        <attribute name="title">
         <string>Mailer settings</string>
        </attribute>
        <layout class="QVBoxLayout" name="loMailerSettings">
         <item>
         <widget class="SDPO::MailerSettingsWidget" name="wMailerSettings" />
         </item>
         </layout>
       </widget>
       <widget class="QWidget" name="pagePagers">
        <attribute name="title">
         <string>Pagers</string>
        </attribute>
        <layout class="QVBoxLayout" name="loPagersOptions">
         <item>
         <widget class="SDPO::PagersOptionsWidget" name="wPagers" />
         </item>
         </layout>
       </widget>
       <widget class="QWidget" name="pageICQ">
        <attribute name="title">
         <string>ICQ</string>
        </attribute>
        <layout class="QVBoxLayout" name="loIcq">
         <item>
         <widget class="SDPO::IcqOptionsWidget" name="wIcq" />
         </item>
         </layout>
       </widget>
       <widget class="QWidget" name="pageJabber">
        <attribute name="title">
         <string>Jabber</string>
        </attribute>
        <layout class="QVBoxLayout" name="loJabber">
         <item>
         <widget class="SDPO::JabberOptionsWidget" name="wJabber" />
         </item>
         </layout>
       </widget>
       <widget class="QWidget" name="pageSmsGsm">
        <attribute name="title">
         <string>SMS: GSM</string>
        </attribute>
        <layout class="QVBoxLayout" name="loSmsGsm">
         <item>
         <widget class="SDPO::SmsGsmOptionsWidget" name="wSmsGsm" />
         </item>
         </layout>
       </widget>
       <widget class="QWidget" name="pageSmsSmpp">
        <attribute name="title">
         <string>SMS: SMPP</string>
        </attribute>
        <layout class="QVBoxLayout" name="loSmsSmpp">
         <item>
         <widget class="SDPO::SmsSmppOptionsWidget" name="wSmsSmpp" />
         </item>
         </layout>
       </widget>
       <widget class="QWidget" name="pagePingTrace">
        <attribute name="title">
         <string>Ping/Trace</string>
        </attribute>
        <layout class="QVBoxLayout" name="loPingTrace">
         <item>
         <widget class="SDPO::PingTraceOptionsWidget" name="wPingTrace" />
         </item>
         </layout>
       </widget>
       <widget class="QWidget" name="pageRci">
        <attribute name="title">
         <string>RCI</string>
        </attribute>
        <layout class="QVBoxLayout" name="loRci">
         <item>
         <widget class="SDPO::RciOptionsWidget" name="wRci" />
         </item>
         </layout>
       </widget>
       <widget class="QWidget" name="pageActiveRma">
        <attribute name="title">
         <string>Active RMA</string>
        </attribute>
        <layout class="QVBoxLayout" name="loActiveRma">
         <item>
         <widget class="SDPO::ActiveRmaServerOptionsWidget" name="wActiveRma" />
         </item>
         </layout>
       </widget>
       <widget class="QWidget" name="pageScheduler">
        <attribute name="title">
         <string>Scheduler</string>
        </attribute>
        <layout class="QVBoxLayout" name="loScheduler">
         <item>
         <widget class="SDPO::SchedulerOptionsWidget" name="wScheduler" />
         </item>
         </layout>
       </widget>
       <widget class="QWidget" name="pageMisc">
        <attribute name="title">
         <string>Misc</string>
        </attribute>
        <layout class="QVBoxLayout" name="loMisc">
         <item>
         <widget class="SDPO::MiscellaneousOptionsWidget" name="wMisc" />
         </item>
         </layout>
       </widget>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="backButton">
         <property name="text">
          <string>&lt;&lt; Back</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="nextButton">
         <property name="text">
          <string>Next &gt;&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QDialogButtonBox" name="FinishCancelHelpBtn">
         <property name="locale">
          <locale language="English" country="UnitedStates"/>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="standardButtons">
          <set>QDialogButtonBox::Cancel|QDialogButtonBox::Help|QDialogButtonBox::Save</set>
         </property>
         <property name="centerButtons">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SDPO::PreferencesWidget</class>
   <extends>QWidget</extends>
   <header>options/qPreferencesWidget.h</header>
  </customwidget>
  <customwidget>
   <class>SDPO::BehaviorWidget</class>
   <extends>QWidget</extends>
   <header>options/qBehaviorWidget.h</header>
  </customwidget>
  <customwidget>
   <class>SDPO::StartupOptionsWidget</class>
   <extends>QWidget</extends>
   <header>options/qStartupOptionsWidget.h</header>
  </customwidget>
  <customwidget>
   <class>SDPO::ServiceOptionsWidget</class>
   <extends>QWidget</extends>
   <header>options/qServiceOptionsWidget.h</header>
  </customwidget>
  <customwidget>
   <class>SDPO::AccessMethodOptionsWidget</class>
   <extends>QWidget</extends>
   <header>options/qAccessMethodOptionsWidget.h</header>
  </customwidget>
  <customwidget>
   <class>SDPO::ProxyOptionsWidget</class>
   <extends>QWidget</extends>
   <header>options/qProxyOptionsWidget.h</header>
  </customwidget>
  <customwidget>
   <class>SDPO::PrimaryLogOptionsWidget</class>
   <extends>QWidget</extends>
   <header>options/qPrimaryLogOptionsWidget.h</header>
  </customwidget>
  <customwidget>
   <class>SDPO::BackupLogOptionsWidget</class>
   <extends>QWidget</extends>
   <header>options/qBackupLogOptionsWidget.h</header>
  </customwidget>
  <customwidget>
   <class>SDPO::SystemLogOptionsWidget</class>
   <extends>QWidget</extends>
   <header>options/qSystemLogOptionsWidget.h</header>
  </customwidget>
  <customwidget>
   <class>SDPO::LogProcessingOptionsWidget</class>
   <extends>QWidget</extends>
   <header>options/qLogProcessingOptionsWidget.h</header>
  </customwidget>
  <customwidget>
   <class>SDPO::HtmlColorsOptionsWidget</class>
   <extends>QWidget</extends>
   <header>options/qHtmlColorsOptionsWidget.h</header>
  </customwidget>
  <customwidget>
   <class>SDPO::ViewersOptionsWidget</class>
   <extends>QWidget</extends>
   <header>options/qViewersOptionsWidget.h</header>
  </customwidget>
  <customwidget>
   <class>SDPO::ReportsOptionsWidget</class>
   <extends>QWidget</extends>
   <header>options/qReportsOptionsWidget.h</header>
  </customwidget>
  <customwidget>
   <class>SDPO::MsgWindowOptionsWidget</class>
   <extends>QWidget</extends>
   <header>options/qMsgWindowOptionsWidget.h</header>
  </customwidget>
  <customwidget>
   <class>SDPO::MailerSettingsWidget</class>
   <extends>QWidget</extends>
   <header>options/qMailerSettingsWidget.h</header>
  </customwidget>
  <customwidget>
   <class>SDPO::PagersOptionsWidget</class>
   <extends>QWidget</extends>
   <header>options/qPagersOptionsWidget.h</header>
  </customwidget>
  <customwidget>
   <class>SDPO::IcqOptionsWidget</class>
   <extends>QWidget</extends>
   <header>options/qIcqOptionsWidget.h</header>
  </customwidget>
  <customwidget>
   <class>SDPO::JabberOptionsWidget</class>
   <extends>QWidget</extends>
   <header>options/qJabberOptionsWidget.h</header>
  </customwidget>
  <customwidget>
   <class>SDPO::SmsGsmOptionsWidget</class>
   <extends>QWidget</extends>
   <header>options/qSmsGsmOptionsWidget.h</header>
  </customwidget>
  <customwidget>
   <class>SDPO::SmsSmppOptionsWidget</class>
   <extends>QWidget</extends>
   <header>options/qSmsSmppOptionsWidget.h</header>
  </customwidget>
  <customwidget>
   <class>SDPO::PingTraceOptionsWidget</class>
   <extends>QWidget</extends>
   <header>options/qPingTraceOptionsWidget.h</header>
  </customwidget>
  <customwidget>
   <class>SDPO::RciOptionsWidget</class>
   <extends>QWidget</extends>
   <header>options/qRciOptionsWidget.h</header>
  </customwidget>
  <customwidget>
   <class>SDPO::ActiveRmaServerOptionsWidget</class>
   <extends>QWidget</extends>
   <header>options/qActiveRmaServerOptionsWidget.h</header>
  </customwidget>
  <customwidget>
   <class>SDPO::SchedulerOptionsWidget</class>
   <extends>QWidget</extends>
   <header>options/qSchedulerOptionsWidget.h</header>
  </customwidget>
  <customwidget>
   <class>SDPO::MiscellaneousOptionsWidget</class>
   <extends>QWidget</extends>
   <header>options/qMiscellaneousOptionsWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../qHostMonitor.qrc"/>
  <include location="../qHostMonitor.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>FinishCancelHelpBtn</sender>
   <signal>accepted()</signal>
   <receiver>OptionsForm</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>FinishCancelHelpBtn</sender>
   <signal>rejected()</signal>
   <receiver>OptionsForm</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
